doc/product_attribute_description.rst

```diff
@@ -0,0 +1,226 @@
+Product Attribute Description
+==============================
+
+* `Introduction`_
+* `Creating a New Product`_
+* `Variants`_
+* `Creating new variants`_
+* `Product Attributes`_
+* `Attribute Sets`_
+* `Attribute Creation`_
+
+.. _Introduction:
+
+Introduction
++++++++++++++
+
+The product module is used for managing all products. This documentation
+describes how to create new products, creation of variants and attributes
+for a product.
```

Comment:
This module is used for adding attributes to products. This documentation describes how to add attributes to variants and show combination of products on webshop.